test/test_linalg.py

```diff
@@ -7453,6 +7454,59 @@ def test_preferred_linalg_library(self):
         self.assertEqual(out_ref, out1.cpu())
         self.assertEqual(out1, out2)
 
+    @onlyCUDA
```

Comment:
You'll probably need `@unittest.skipIf(not CUDA11OrLater, "Only CUDA 11+ is supported")` unless you want to fix CUDA-10.2 bugs